Source/WTF/wtf/PlatformHave.h

```diff
@@ -1469,6 +1469,15 @@
 #define HAVE_NSVIEW_CLIPSTOBOUNDS_API 1
 #endif
 
+#if (PLATFORM(MAC) && __MAC_OS_X_VERSION_MIN_REQUIRED >= 140000) \
+    || (PLATFORM(MACCATALYST) && __IPHONE_OS_VERSION_MIN_REQUIRED >= 170000) \
+    || (PLATFORM(IOS) && __IPHONE_OS_VERSION_MIN_REQUIRED >= 170000) \
```

Comment:
Do we need PLATFORM(VISION)? Should it be PLATFORM(IOS_FAMILY)?

Reply:
This is not required for visionOS at the moment, I believe.

Thanks for reviewing!